docs/generated/sparse.COO.rst

```diff
@@ -0,0 +1,110 @@
+COO
+===
+
+.. currentmodule:: sparse
+
+.. autoclass:: COO
+
+   
+   
+   .. rubric:: Attributes
+   .. autosummary::
+      :toctree:
+   
+      COO.T
+   
```

Comment:
Can we remove extra endlines here?

Reply:
These are auto-generated. It's either ugliness in here or in the files that generate them.

Reply:
OK, if they're auto-generated then I remove my comment